src/Uno.UI/Helpers/UnoRuntimePlatform.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Uno.UI.Helpers;
+
+public enum UnoRuntimePlatform
+{
+	Android,
+	iOS,
+	MacCatalyst,
+	MacOSX,
```

Comment:
I think this entry is not necessary, as it is actually only temporary anyway, as we will be dropping the legacy macOS implementation